src/main/java/Deadline.java

```diff
@@ -5,14 +5,14 @@
     protected String by;
 
     public Deadline(String description, String by) {
```

Comment:
Should it be Deadline extends Task? or is it ok to just write Deadline.

Reply:
Class Deadline extends Class Task so that it could override the method toString() and inherit variables from Class Task